src/config.ts

```diff
@@ -544,6 +545,8 @@ export class KubeConfig implements SecurityAuthentication {
             } else {
                 throw new Error('Unsupported proxy type');
             }
+        } else if (cluster?.server?.startsWith('http:')) {
```

Comment:
Can we also check for `skipTlsVerify` being `true` before we allow http URLs? It's really not secure to do this.

Reply:
Pushed a commit to check for `skipTlsVerify` as well and added a test for this.
I agree that it's not secure but I think the library should allow this as it is possible to do. 🤔